.circleci/config.yml

```diff
@@ -163,63 +176,29 @@ commands:
           keys:
             - build-cache-services-master-
 
-  run_command:
-    parameters:
-      command_description:
-        type: string
-      command_name:
-        type: string
-    description: Runs commands based on passed parameters
-    steps:
-      - run:
-          name: << parameters.command_description >>
-          command: |
-            if [ "${RUN_ALL}" == "true" ];
-            then
-              yarn << parameters.command_name >>
-            else
-              yarn << parameters.command_name >> --since master...HEAD
-            fi
-
-  cache_checksum:
-    description: Yarn node_modules cache checksum
-    steps:
-      - run:
-          name: Create cache checksum file
-          command: |
-            mkdir -p ~/.tmp/checksumfiles
-            find . -type f -name 'yarn.lock' -not -path "*node_modules*" -exec cat {} + >> ~/.tmp/checksumfiles/yarn.lock
-
-  restore_node_modules_cache:
-    description: Restore node_modules folder
-    steps:
-      - restore_cache:
-          name: Restore node_modules
-          keys:
-            - yarn-dependencies-{{ .Branch }}-{{ checksum "~/.tmp/checksumfiles/yarn.lock" }}
-
   build_packages:
     description: Locally builds all packages in the monorepo
     steps:
-      - restore_cache_dist_directories
       - run_command:
           command_description: Build packages
-          command_name: "build"
-      - save_cache_dist_directories
+          command_name: build 
 
   lint_packages:
     description: Locally lints all packages in the monorepo
     steps:
       - run_command:
           command_description: Run linting for packages
-          command_name: "lint"
+          command_name: lint
 
   unit_integration_tests:
     description: Run Unit / Integration Tests
+    parameters:
+      component_type:
+        type: string
```

Comment:
Added a new `component_type` parameter (atoms / molecules / organisms / tools / services`) which is passed down to the command to determine which package.json script to run.

This will result in something like `yarn ci:test:<component_type>` being executed.